README.md

```diff
@@ -1,4 +1,2 @@
 # covid-19
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/db1476eb3cc745d8b2b82b762228c0b6)](https://app.codacy.com/gh/azaria-dedmon/covid-19?utm_source=github.com&utm_medium=referral&utm_content=azaria-dedmon/covid-19&utm_campaign=Badge_Grade_Settings)
-
-[![Coverage Status](https://coveralls.io/repos/github/azaria-dedmon/covid-19/badge.svg?branch=master)](https://coveralls.io/github/azaria-dedmon/covid-19?branch=master)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/db1476eb3cc745d8b2b82b762228c0b6)](https://app.codacy.com/gh/azaria-dedmon/covid-19?utm_source=github.com&utm_medium=referral&utm_content=azaria-dedmon/covid-19&utm_campaign=Badge_Grade_Settings)[![Coverage Status](https://coveralls.io/repos/github/azaria-dedmon/covid-19/badge.svg?branch=master)](https://coveralls.io/github/azaria-dedmon/covid-19?branch=master)[![Maintainability](https://api.codeclimate.com/v1/badges/939cec24acf41c03c578/maintainability)](https://codeclimate.com/github/azaria-dedmon/covid-19/maintainability)
```

Comment:
@azaria-dedmon In the readme, have each badges `markup/code` on a different line but without spacing. This will keep the badges from being clogged up next to one another.